r/R/utilities_expression.R

```diff
@@ -0,0 +1,62 @@
+#' Returns expression values for selected genes
+#'
+#' @param genes - Must have names and input(ensmbl ids)
+#' @param data
+#'
+#' @return
+#' @export
+#'
+#' @examples
+getExpressionValues <- function(genes, data) {
+  quantile_threshold <- 0.95
+  #
+  # Get the expression values for those genes in the corresponding matrix.
+  geneExpression <- data@assays$RNA@data[unique(genes$input), , drop = FALSE]
+  geneExpression <- as.data.frame(t(as.matrix(geneExpression)))
+  geneExpression$cells_id <- data@meta.data$cells_id
+  geneExpression <- geneExpression[order(geneExpression$cells_id), ]
+  geneExpression <- geneExpression %>%
+    tidyr::complete(cells_id = seq(0, max(data@meta.data$cells_id))) %>%
+    dplyr::select(-cells_id) %>%
+    as.data.frame()
+  # worried about duplicate gene row.names in @data
```

Comment:
what do you want to say with this comment?

Reply:
I think it's a remnant from the past.

Reply:
can you remove it in this case? Or rephrase it.

Reply:
Yes, I've pushed the commit :)